### ISSUE TITLE
Only one FilesInterceptor configuration per Controller possible

### ISSUE THREAD

**Issue by kiwikern** [nestjs/nest#935]
## I'm submitting a...
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When two `FilesInterceptor`s (multiple files!) are defined within one controller, only the last configuration options are applied. So when for the two endpoints two different fields are defined with `@UseInterceptors(FilesInterceptor('first'))` and `@UseInterceptors(FilesInterceptor('second'))`, both will only accept `second`. It also doesn't work for a `MulterOptions` object like
```
@UseInterceptors(FilesInterceptor('first', null, {
    dest: './uploads/first'
  }))
```

It does work however for `FileInterceptor`s (single file!) or when two different `Controller` are defined.

## Expected behavior

The given configuration should be applied.

## Minimal reproduction of the problem with instructions
Create a Controller with those two endpoints:
```
  @Post('first')
  @UseInterceptors(FilesInterceptor('first'))
  async uploadFirst(@UploadedFiles() files) {
    return files;
  }

  @Post('second')
  @UseInterceptors(FilesInterceptor('second'))
  async uploadSecond(@UploadedFiles() files) {
    return files;
  }
```

Make the following form-data POST requests to the given endpoint with a file in the given field:

| Endpoint | Field | Result | Expected |
| --- | --- | --- | --- |
| `first` | `first` | `400 - Unexpected field` | Should work ❌ |
| `first` | `second` | `200` | Should not work ❌ |
| `second` | `second` | `200` | Should work ✔️ |

It might be the other way around: first works, second doesn't.
## What is the motivation / use case for changing the behavior?

Came up in this [Stackoverflow question](https://stackoverflow.com/q/51635203/4694994).

## Environment

<pre><code>
Nest version: 5.1.2

 
For Tooling issues:
- Node version: v8.10.0 
- Platform: Windows  

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Fixed in `v5.2.0`. Will publish soon

**Comment by kamilmysliwiec** [comment]
5.2.0 is published!
